aspnetcore/release-notes/aspnetcore-6.0.md

```diff
@@ -0,0 +1,704 @@
+
+When implementing this interface, use the `DisposeAsync` method to release resources.
+
+Consider a controller that creates and uses a <xref:System.Text.Json.Utf8JsonWriter>. `Utf8JsonWriter` is an `IAsyncDisposable` resource:
+
+[!code-csharp[](aspnetcore-6.0/samples/HomeController.cs?name=snippet&highlight=1,9)]
+
+`IAsyncDisposable` must implement `DisposeAsync`:
+
+[!code-csharp[](aspnetcore-6.0/samples/HomeController.cs?name=snippet2)]
+
+The preceding code:
+
+* Calls `Dispose` to release other unmanaged resources.
+* Tells the garbage collector not to run.
+* Awaits `DisposeAsyncCore`, shown in the following code:
+
+[!code-csharp[](aspnetcore-6.0/samples/HomeController.cs?name=snippet3)]
+
+### Vcpkg port for SignalR C++ client
+
+[Vcpkg](https://github.com/microsoft/vcpkg) is a cross-platform command-line package manager for C and C++ libraries. We’ve recently added a port to `vcpkg` to add `CMake` native support for the SignalR C++ client. `vcpkg` also works with MSBuild.
+
+The SignalR client can be added to a CMake project with the following snippet when the vcpkg is included in the toolchain file:
+
+```dotnetcli
+find_package(microsoft-signalr CONFIG REQUIRED)
+link_libraries(microsoft-signalr::microsoft-signalr)
+```
+
+With the preceding snippet, the SignalR C++ client is ready to use [`#include`](https://github.com/halter73/SignalR-Client-Cpp-Sample/blob/main/sample.cpp#L3-L5) and used in a project without any additional configuration. For a complete example of a C++ application that utilizes the SignalR C++ client, see the [halter73/SignalR-Client-Cpp-Sample](https://github.com/halter73/SignalR-Client-Cpp-Sample) repository.
+
+<!-- end of # signalR -->
+
+## Blazor
+
+### Blazor WebAssembly native dependencies support
+
+Blazor WebAssembly apps can use native dependencies built to run on WebAssembly. For more information, see <xref:blazor/host-and-deploy/webassembly?view=aspnetcore-6.0#native-dependencies-support>.
+
+### WebAssembly Ahead-of-time (AOT) compilation and runtime relinking
+
+Blazor WebAssembly supports ahead-of-time (AOT) compilation, where you can compile your .NET code directly into WebAssembly. AOT compilation results in runtime performance improvements at the expense of a larger app size. Relinking the .NET WebAssembly runtime trims unused runtime code and thus improves download speed. For more information, see [Ahead-of-time (AOT) compilation](xref:blazor/host-and-deploy/webassembly?view=aspnetcore-6.0#ahead-of-time-aot-compilation) and [Runtime Relinking](xref:blazor/host-and-deploy/webassembly?view=aspnetcore-6.0#runtime-relinking).
+
+### Persist prerendering state
+
+Blazor supports persisting state in a prerendered page so that the state doesn’t need to be recreated when the app is fully loaded. For more information, see <xref:blazor/components/prerendering-and-integration?view=aspnetcore-6.0&pivots=server#preserve-prerendered-state>.
```

Comment:
```suggestion
Blazor supports persisting state in a prerendered page so that the state doesn't need to be recreated when the app is fully loaded. For more information, see <xref:blazor/components/prerendering-and-integration?view=aspnetcore-6.0&pivots=server#preserve-prerendered-state>.
```